tools/make-v8.sh

```diff
@@ -1,38 +1,53 @@
 #!/bin/bash
 
-
-git_origin=$(git config --get remote.origin.url | sed 's/.\+[\/:]\([^\/]\+\/[^\/]\+\)$/\1/')
-git_branch=$(git rev-parse --abbrev-ref HEAD)
-v8ver=${1:-v8} #default v8
-svn_prefix=https://github.com
-svn_path="$svn_prefix/$git_origin/branches/$git_branch/deps/$v8ver"
-#svn_path="$git_origin/branches/$git_branch/deps/$v8ver"
-gclient_string="solutions = [{'name': 'v8', 'url': '$svn_path', 'managed': False}]"
+if [ $# -eq 0 ]; then
+  # Default branch is master
+  BRANCH="master"
+else
+  # eg: 5.4-lkgr
+  BRANCH="$1"
+fi
 
 # clean up if someone presses ctrl-c
 trap cleanup INT
 
 function cleanup() {
   trap - INT
-
   rm .gclient || true
   rm .gclient_entries || true
   rm -rf _bad_scm/ || true
-
-  #if v8ver isn't v8, move the v8 folders
-  #back to what they were
-  if [ "$v8ver" != "v8" ]; then
-    mv v8 $v8ver
-    mv .v8old v8
+  if [ "$BRANCH" == "master" ]; then
+    echo "git cleanup if branch is master"
```

Comment:
Launching a job with somebody's pr should still work, since their git repo will be tracking the files they updated. Like the job Ben started with my changes https://ci.nodejs.org/job/node-test-commit-v8-linux/390/
Regarding BRANCH, yes it indeed meant to test specific v8 version if needed, otherwise it will get the tools from V8 master. I might have misunderstood the exact purpose of this job, as you mentioned in the other comment if the purpose is to get the tools of a specific v8 version then I will have to do the cleanup for any branch. Will make the necessary changes.